ui/raidboss/data/02-arr/dungeon/haukke_manor.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.HaukkeManor,
+  triggers: [
+    {
+      // Dark Mist Stun with Suppression for duplicate messages
+      id: 'Haukke Normal Dark Mist Stun',
+      netRegex: NetRegexes.startsUsing({ id: '2C1', source: ['Manor Maidservant', 'Manor Claviger', 'Lady Amandine'] }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      suppressSeconds: 2,
+      response: Responses.stun('info'),
+    },
+    {
+      // Soul Drain
+      id: 'Haukke Normal Steward Soul Drain Stun',
+      netRegex: NetRegexes.startsUsing({ id: '35C', source: 'Manor Steward' }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      response: Responses.stun('info'),
+    },
+    {
+      // Amandine Specific Dark Mist for Classes Without Stuns
+      // Particle and spell effects in the room can make this Dark Mist hard to see.
+      // Callout ensures dodge.
+      id: 'Haukke Normal Amandine Dark Mist Dodge',
+      netRegex: NetRegexes.startsUsing({ id: '2C1', source: 'Lady Amandine' }),
+      condition: function(data, matches) {
+        return !data.CanStun();
+      },
+      response: Responses.outOfMelee('alert'),
+    },
+    {
+      // Void Fire 3
```

Comment:
One last tiny nit here is that I don't think there's a need for these comments here when the `id` also explains the ability name.  Sometimes it can be ambiguous and comments are great, but in these cases I think the information is redundant.